tikv-configuration-file.md

```diff
@@ -850,6 +856,20 @@ coprocessor 相关的配置项。
 + 分裂后新 Region 的 key 的个数，此值属于估算值。
 + 默认值：960000
 
+### `enable-region-bucket`
+
++ 是否开启 region bucket 将 region 划分为更小的的区间。
++ 默认值：false
+> **警告：**
+>
+> - region buckets 是 TiDB 在 v6.1.0 中引入的实验特性，不建议在生产环境中使用。
+> - 这个参数只有在将 region size 调到 128MiB 及以上时才有意义。将 region size 调大可能会有潜在的性能回退、搬迁缓慢的风险。
```

Comment:
Is it ignored when region-bucket-size >= region-split-size? Or region-bucket-size >= 128MB?

Reply:
The parameter takes affect only when bucket is enabled. If `region-bucket-size >= region-split-size`, it will not split the region into smaller trunk until the actual sizes exceeds the limit.

Reply:
This seems to contradict `这个参数只有在将 region size 调到 128MiB 及以上时才有意义。`, rest looks good.

Reply:
Because it will not split the region into smaller trunk until actual sizes exceeds the limit, so it's pointless to set region size less than 128MiB as region size will not ever reach the bucket size.

Reply:
Why is that? Default bucket size seems to be `96MB`, and can be configured to be smaller.